anchore_engine/db/db_catalog_image_docker.py

```diff
@@ -242,7 +242,7 @@ def get_tag_histories(session, userId, registries=None, repositories=None, tags=
                 CatalogImageDocker.imageDigest == CatalogImage.imageDigest,
             ),
         )
-        .filter(CatalogImage.userId == userId)
+        .filter(CatalogImage.userId == userId, CatalogImage.analyzed_at.isnot(None))
```

Comment:
Looking downstream at this, the analysis_status is handled in the actual archiver task. So lets just handle this case in the timestamp comparison code instead of filtering at the db query.